src/components/MyProfile.js

```diff
@@ -0,0 +1,45 @@
+import { useSelector } from 'react-redux';
+import '../style/MyProfile.css';
+
+const MyProfile = () => {
+  const myMissions = useSelector((state) => state.missions);
+  const myDragons = useSelector((state) => state.dragons);
+  const rockets = useSelector((state) => state.RocketsReducer);
+
+  return (
+    <div className="my-profile__container">
+      <div className="my-profile__rockets">
+        <h2>My Rockets</h2>
+        {rockets.map((rocket) => (
+          rocket.reserved && (
+          <div key={rocket.id}>
+            <p>{rocket.rocket_name}</p>
+          </div>
+          )
+        ))}
+      </div>
+      <div className="my-profile__missions">
+        <h2>My Missions</h2>
+        {myMissions.map((mission) => (
+          mission.reserved && (
+            <div key={mission.mission_name}>
+              <p>{mission.mission_name}</p>
+            </div>
+          )
+        ))}
+      </div>
+      <div className="my-profile__dragons">
+        <h2>My Dragons</h2>
+        {myDragons.map((dragon) => (
+          dragon.reserved && (
+            <div key={dragon.id}>
+              <p>{dragon.name}</p>
+            </div>
+          )
```

Comment:
- [ ] Your reserved Rockets, Missions, and Dragons should be rendered using the `filter` method. Kindly do that.

  **SUGGESTION** You can consider filtering the `myMissions`, `myDragons`, `rockets` variables before mapping. 

  ![reem-lab-filter](https://user-images.githubusercontent.com/36542986/156488139-75e011d7-f8df-4c50-b70c-dab954f9d5a1.jpg)